### PR TITLE
Add Danger 

### PR DESCRIPTION
**Pull request by ReadmeCritic**
This adds https://danger.systems to `awesome-hyperterm` to verify the capitalization of **HyperTerm** in a pull request (Related to #34). 

@bnb if you want to proceed, we have to configure the `API_TOKEN` in Travis CI, let me know.

### CODE REVIEW

File: Danger file [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Ensure there is a summary for a pull request
+#fail 'Please provide a summary in the Pull Request description' if github.pr_body.length < 5
+
+# Warn if pull request is not updated
+#warn 'Please update the Pull Request title' if github.pr_title.include? 'Update README.md'
+
+# Verifies capitalization of HyperTerm
+fail 'Please provide a summary in the Pull Request description' if github.pr_body.include? 'Hyperterm'
```
Comment (bnb) [inline Danger file]: This doesn't seem like the correct description for the check.

**Comment by ReadmeCritic** [comment]
@bnb I will email you some instructions about CI and the `API_TOKEN`

**Comment by bnb** [comment]
Hey. Sorry to do this (you've done a lot of work), but I don't really fully understand what this code is doing. I don't know Ruby at all, so that's a big barrier to this - if it were JS or Node, I'd be able to consume it, but at this point I'm not comfortable with adding Danger because I can't read the code in a meaningful way. Additionally, it seems extremely inappropriate to be sent a GitHub API Key randomly and put it in my CI.

Thanks, but I'm going to close this and not accept further PRs on this specific topic. I don't mind running a find/replace locally if I really think it's needed. CI for basic spelling seems slightly overkill.

**Comment by ReadmeCritic** [comment]
No worries
But the idea if you read https://danger.systems/ is for a custom report to run on each pull request

Here it would fail with a message about the capilization of HyperTerm

Let me know if you change your mind

**Comment by matheuss** [comment]
🤔
screenshot_20160730-202941

**Comment by bnb** [comment]
@matheuss That's because ReadmeCritic keeps linking the HTTPS version of the site, when the site doesn't actually support HTTPS.

**Comment by ReadmeCritic** [comment]
https://github.com/danger/danger

**Pull request closed without merging**